ccd-definition/ComplexTypes/CaseToPostLRspec.json

```diff
@@ -0,0 +1,37 @@
+[
+  {
+    "ID": "CaseToPostLRspec",
```

Comment:
Just a minor thing, I am not able to understand what is this variable for if I don't know that this is for Pin in Post.
My suggestion would be to use name like DefendantPinToPostLRSpec or something similar.

Reply:
Changed the name to DefendantPinToPostLRSpec as suggested